client/ayon_core/modules/deadline/repository/custom/plugins/GlobalJobPreLoad.py

```diff
@@ -471,12 +471,21 @@ def inject_ayon_environment(deadlinePlugin):
         ]
 
         add_kwargs = {
-            "project": job.GetJobEnvironmentKeyValue("AVALON_PROJECT"),
-            "asset": job.GetJobEnvironmentKeyValue("AVALON_ASSET"),
-            "task": job.GetJobEnvironmentKeyValue("AVALON_TASK"),
-            "app": job.GetJobEnvironmentKeyValue("AVALON_APP_NAME"),
             "envgroup": "farm",
         }
+        # Support backwards compatible keys
+        for key, env_keys in (
+            ("project", ["AYON_PROJECT_NAME", "AVALON_PROJECT"]),
+            ("asset", ["AYON_FOLDER_PATH", "AVALON_ASSET"],
```

Comment:
There's a typo here.
```suggestion
            ("asset", ["AYON_FOLDER_PATH", "AVALON_ASSET"]),
```